packages/qtbase.rb

```diff
@@ -50,7 +44,8 @@ def self.build
            '-system-zlib',
            '-system-libpng',
            '-system-libjpeg',
-           '-system-freetype'
+           '-system-freetype',
+           '-no-openssl'
```

Comment:
I'm hesitant to remove openssl.  I'm sure we can figure out a way to include it either by updating libressl or reverting back to openssl.

Reply:
Totally agree. But I had to remove it to make the other thing work. Updating libressl or something like it is a huge work due to its impact on the chromebrew basis. As said in a related issue, I chose to remove it in a first step, then state on libressl vs openssl and their updates and finally reactivate it in qtbase

Reply:
Fair enough.  I'm glad you were able to compile these qt packages successfully because I couldn't do it.